cantor-archive/src/main/java/com/salesforce/cantor/archive/EventsArchiverOnFile.java

```diff
@@ -100,4 +117,24 @@ public static void restore(final Events events, final String namespace, final Pa
                     totalEventsRestored, namespace, archiveFile, TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startNanos));
         }
     }
+
+    @Override
+    public boolean hasArchives(final String namespace, final long startTimestampMillis, final long endTimestampMillis) {
```

Comment:
Didn't we remove these from the interface?

Reply:
Yes, problem with having so many PRs up! This slipped by XD Removed.